deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -233,6 +233,16 @@ data:
         </buffer>
       </match>
     </label>
+  logs.source.generic.conf: |-
```

Comment:
In order for this pipeline to take effect you'd have to add `logs.source.generic.conf` to `logs.conf`

Reply:
ah, missed that. Thanks